SwitchTrigger4/src/switchtrigger4.cpp

```diff
@@ -63,6 +116,20 @@ const LV2_Descriptor* lv2_descriptor(uint32_t index)
 LV2_Handle SwitchTrigger::instantiate(const LV2_Descriptor* descriptor, double samplerate, const char* bundle_path, const LV2_Feature* const* features)
 {
     SwitchTrigger *plugin = new SwitchTrigger();
+
+    for(int i=0; features[i]; i++)
+    {
+        if(!strcmp(features[i]->URI,LV2_URID__map))
+        {
+            plugin->urid_map = (LV2_URID_Map *) features[i]->data;
+            if(plugin->urid_map)
+            {
+                plugin->URIDs.atom_Path = plugin->urid_map->map(plugin->urid_map->handle,LV2_ATOM__Path);
+                plugin->URIDs.switch_channel = plugin->urid_map->map(plugin->urid_map->handle,CHANNEL_URI);
+            }
+        }
```

Comment:
add a break after handling urid map, as there is no need to check any more features afterwards